third-party/realdds/include/realdds/topics/notification/notification-msg.h

```diff
@@ -30,28 +34,30 @@ namespace topics {
 
     
 namespace raw {
-namespace device {
 class notificationPubSubType;
-class notification;
-}  // namespace device
 }  // namespace raw
 
 
-namespace device {
-
-    
 class notification
 {
 public:
-    using type = raw::device::notificationPubSubType;
+    using type = raw::notificationPubSubType;
+
+    enum class data_type
+    {
+        // Note: these need to be kept in sync with the raw version
+        JSON = raw::NOTIFICATION_DATA_JSON,
+        CBOR = raw::NOTIFICATION_DATA_CBOR,
+        CUSTOM = raw::NOTIFICATION_DATA_CUSTOM,
```

Comment:
What does custom means?
CBOR can google :)

Reply:
Custom user data -- not for interpretation by the `notification` code.